cmd/scollector/collectors/httpunit.go

```diff
@@ -10,33 +10,34 @@ import (
 	"github.com/StackExchange/httpunit"
 )
 
-func HTTPUnitTOML(filename string) error {
+func HTTPUnitTOML(filename string, freq time.Duration) error {
 	var plans httpunit.Plans
 	if _, err := toml.DecodeFile(filename, &plans); err != nil {
 		return err
 	}
-	HTTPUnitPlans(filename, &plans)
+	HTTPUnitPlans(filename, &plans, freq)
 	return nil
 }
 
-func HTTPUnitHiera(filename string) error {
+func HTTPUnitHiera(filename string, freq time.Duration) error {
 	plans, err := httpunit.ExtractHiera(filename)
 	if err != nil {
 		return err
 	}
-	HTTPUnitPlans(filename, &httpunit.Plans{
-		Plans: plans,
-	})
+	HTTPUnitPlans(filename, &httpunit.Plans{Plans: plans}, freq)
 	return nil
 }
 
-func HTTPUnitPlans(name string, plans *httpunit.Plans) {
+func HTTPUnitPlans(name string, plans *httpunit.Plans, freq time.Duration) {
+	if freq < time.Second {
```

Comment:
this seems a bit odd to me. If they didn't set it, and it was zero, I would understand. But if they set `15` expecting it to be seconds or minutes or who knows what, 5 mins is an odd default. Maybe error with a helpful message?